src/Combine.elm

```diff
@@ -231,12 +231,12 @@ many1 p =
 
 sepBy : Parser s x -> Parser s a -> Parser s (List a)
 sepBy sep p =
-    or (sepBy1 sep p) (succeed [])
+    or (sepBy1 sep p |> map (\( head, rest ) -> head :: rest)) (succeed [])
 
 
-sepBy1 : Parser s x -> Parser s a -> Parser s (List a)
+sepBy1 : Parser s x -> Parser s a -> Parser s ( a, List a )
```

Comment:
I've modified sepBy1 to return a nonempty list rather than lose this information. This has helped make it clear which other datastructures could also use nonempty lists.